### DISCUSSION TITLE
BUG: Page filled with stream of irrelevant posts

### DISCUSSION THREAD

**Discussion by csilverman** [community/community#54111] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

The majority of the screen is taken up with something that looks like an attempt at Twitter, except even less useful. This is clearly a malfunction, since I can't imagine any thinking person at GitHub would honestly believe this was an improvement to the site.

How do I fix this? Do I need to clear my cookies or something? Happening in all browsers.

**Comment by TomArrow** [comment]
Honestly astonishing that someone thought this was useful LOL.

**Reply by csilverman** [reply]
I'm just imagining the meeting where some boy genius at Microsoft managed to convince everybody that "LinkedIn-izing GitHub" would be "a disruptive paradigm shift".
